docs/testing/debug-desktop-using-edge-chromium.md

```diff
@@ -78,45 +78,85 @@ These instructions assume you have experience using the command line, understand
 >
 > If debugging stops working; for example, if breakpoints are being ignored; stop debugging. Then, if necessary, close all host application windows and the Node window. Finally, close Visual Studio Code and reopen it.
 
-### Appendix A
+## Debug a project not created with Yo Office
 
 If your project wasn't created with Yo Office, you need to create a debug configuration for Visual Studio Code.
 
+### Configure package.json file
+
+1. Ensure you have a `package.json` file. If you don't already have a package.json file, run `npm init` in the root folder of your project and answer the prompts.
+1. Run `npm install office-addin-debugging`. This package will sideload your add-in for debugging.
```

Comment:
Nit. Personal preference to avoid future tense in step-by-step instructions.
```suggestion
1. Run `npm install office-addin-debugging`. This package sideloads your add-in for debugging.
```